python/ray/__init__.py

```diff
@@ -106,6 +108,8 @@ def _configure_system():
     "actor",
     "available_resources",
     "cancel",
+    "client",
+    "ClientBuilder",
```

Comment:
can we remove this?

Reply:
the "ClientBuilder" part yes, "client", no.

The `__all__` in python is used for `import * from ray`

Reply:
See my comment above.